website/docs/limitations.md

```diff
@@ -0,0 +1,28 @@
+---
+id: limitations
+title: Limitations
+description: A limitations table where every technical and business limitation is collected for each subscription plan.
+---
+
+## A limitations table where every technical and business limitation is collected for each subscription plan.
+
+:::info Every bolded value in the table marks a technical limitation. Meaning that the value can be increased by [contacting us](https://configcat.com/support/).
```

Comment:
I'd go with a * in the table instead of bolding the tech limits. This way it seems that those are the important ones.
Or it would be good to show it somehow that those are unlimited, but has tech limits.